viewing_party/party.py

```diff
@@ -1,23 +1,181 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_info = {}
+    if not title or not genre or not rating:
+        return None

```

Comment:
This is a great guard clause! It allows your function to resume its normal process if the guard clause doesn't catch anything out of the ordinary.